generators/typescript-v2/ast/src/typescript.ts

```diff
@@ -12,6 +12,27 @@ export function reference(arg: AST.Reference.Args): AST.Reference {
     return new AST.Reference(arg);
 }
 
+export function func(args: AST.Func.Args): AST.Func {
+    return new AST.Func(args);
+}
+
+export function method(args: AST.Method.Args): AST.Method {
+    return new AST.Method(args);
+}
+
+export function invokeFunction(args: AST.FunctionInvocation.Args): AST.FunctionInvocation {
+    return new AST.FunctionInvocation(args);
+}
+
+export function invokeMethod(args: AST.MethodInvocation.Args): AST.MethodInvocation {
+    return new AST.MethodInvocation(args);
+}
+
+export function parameter(args: AST.Parameter.Args): AST.Parameter {
+    const foo = AST.Parameter;
```

Comment:
What's this used for?

Reply:
I think that was a debug breakpoint at one point